cicd-scripts/run-e2e-tests.sh

```diff
@@ -16,11 +16,12 @@ fi
 cd ${WORKDIR}/..
 git clone https://github.com/open-cluster-management/observability-e2e-test.git
 cd observability-e2e-test
+git checkout release-2.2
```

Comment:
next time, you can use `git clone --depth 1 -b release-2.2 https://github.com/open-cluster-management/observability-e2e-test.git`.

Reply:
got it, thanks